_posts/2020-09-21-permission-editor.md

```diff
@@ -0,0 +1,111 @@
+---
+
+title: "Permission editor on the admin ui"
+author: [pusztaienike]
+image: "../img/posts/permission_editor.jpg"
+tags: [sensenet, permission]
+
+---
+
+"Nobody can hurt me without my permission. /Mahatma Gandhi/"
+
+---
+
+In the IT world you often handle sensitive data, which you don't want to share with the whole universe. But how you can hide those contents from outsiders? What if you would like to share those things only with a specific group of users? Or other data with another group? 
+
+Of course there is a common solution for that: permission system. Almost every software has permissions which handle what users can see or do.
+
+I think most of u use some kind of social media. Think about searching for someone and that by default you cannot see her/his photos, but after you apply a friend request everything changes.
+And what about your company data? Unless you are the boss, I am sure that your company has data for which you do not have sufficient permissions.
+
+Permissions are everywhere. And of course sensenet has its own permission system as well. On the new admin surface 
+handling permissions was only possible with coding and running commands. Till now. 🕛 In the last couple of weeks we were about to design and implement the simplest, easiest to use **permission editor**.
+
+## Customize it as you want 🔧🔨
+
+sensenet has a lot of built-in permissions but there are some reserved one that you are free to use in case of you have special needs ([custom permissions](https://docs.sensenet.com/concepts/user-and-permission-management/04-custom-roles-and-permissions#permissions)). You can also group these permissions for the permission editor dialog as you want. The related settings file is available under Setup menu / Permission.settings.
```

Comment:
```suggestion
sensenet has a lot of built-in permissions but there are some reserved ones that you are free to use in case of you have special needs ([custom permissions](https://docs.sensenet.com/concepts/user-and-permission-management/04-custom-roles-and-permissions#permissions)). You can also group these permissions for the permission editor dialog as you want. The related settings file is available under Setup menu / Permission.settings.
```